mern-ecommerce/routes/order.js

```diff
@@ -29,15 +29,15 @@ router.get(
   requireSignin,
   isAuth,
   isAdmin,
-  getStatusValues
+  getStatus
```

Comment:
The function `getStatus` is not defined in the provided code snippet. It is being called in the `router.get` method in the `order.js` file, but its implementation is not provided. 

Please define the `getStatus` function or provide its implementation to review it correctly.